charms/jupyter-ui/requirements-fmt.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --resolver=backtracking requirements-fmt.in
+#    pip-compile requirements-fmt.in
```

Comment:
Looks like `pip-compile` is being executed different here than how renovate executes it.  If we commit this, renovate will immediately change it back (see PRs like [here](https://github.com/canonical/kubeflow-dashboard-operator/pull/78/files#diff-e76170d7f31d3ddd4cfd6059a65d86ef502d580f4f6476d44ea74ad4d31aa6dcL5)).  This comment can be applied to the headers in all the `.in` files

Reply:
It looks like in other `requirements*.txt` files `pip-compile` was involked without `--resolver=backtracking`. So, we have some inconsistency here. 
How should we do it: with or without ` --resolver=backtracking`?